407f738

commit 407f738
Author: Alain Delree <alain.delree@gmail.com>
Date:   Tue Aug 25 12:27:12 2026 +0200

    Corrige la faille .gitignore sur les fichiers .env

diff --git a/.gitignore b/.gitignore
index 4209c0a..c5c8993 100644
--- a/.gitignore
+++ b/.gitignore
@@ -22,7 +22,8 @@ staticfiles/
 local_settings.py
 
 # Variables d'environnement
-.env.local
+.env*
+!.env.example
 
 # Éditeurs
 .vscode/
